py3-setuptools.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: py3-setuptools
+  version: 70.0.0
+  epoch: 1
+  description: Easily download, build, install, upgrade, and uninstall Python packages
+  copyright:
+    - license: "MIT"
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: setuptools
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-python
+      - wolfi-base
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: f211a66637b8fa059bb28183da127d4e86396c991a942b028c6650d4319c3fd0
+      uri: https://files.pythonhosted.org/packages/source/s/setuptools/setuptools-${{package.version}}.tar.gz
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/pip-build-install-bootstrap
+        with:
+          python: python${{range.key}}
+      - uses: strip
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+
+update:
```

Comment:
I think this is picking up beta relases 

```
py3-pip: update found newer version 24.1b1 compared with package.version 24.0 in melange config
```

Reply:
actually per https://packaging.python.org/en/latest/specifications/version-specifiers/#public-version-identifiers exclude alpha, beta, rc and dev 

```
{a|b|rc}N
.devN
```

Reply:
Pushed just the `b` exclude 24.1b for now and I'll look at making `wolfictl check update` support semver pre-release more broadly

https://semver.org/spec/v2.0.0.html#spec-item-9

https://semver.org/spec/v2.0.0.html#backusnaur-form-grammar-for-valid-semver-versions